tests/test_functions.py

```diff
@@ -619,3 +619,20 @@ def test_round_to_nearest_half(dataframe):
     assert df.iloc[6, 1] == 1.0
     assert df.iloc[7, 1] == 2.5
     assert df.iloc[8, 1] == 3.0
+
+
+def test_make_currency_api_request(dataframe):
+    """This test assumes that the GBP will always be more than the USD."""
+
+    df = dataframe.convert_currency("a", "USD", "GBP")
+
+    assert df.iloc[0, 0] < 1
+
+
+def test_currency_makes_column_decrease(dataframe):
+    """This test assumes that the GBP will always be more than the USD."""
```

Comment:
This test likewise makes a very large assumption about exchange rates, one that cannot be guaranteed to be true all the time. I'd suggest removing this one, or replacing it with a test that checks the dtype of the resultant column.